guides/_posts/2014-01-5-vfl.html

```diff
@@ -88,29 +100,29 @@
 
```

Comment:
In case it's hidden, it's the line that reads "If no `gap()` is defined, but a hyphen is used to make a connection then a `standard gap` variable is created,"